testing/testcontainers/java/README.rst

```diff
@@ -0,0 +1,63 @@
+#######################
+Testcontainers for Java
+#######################
+
+*How to run integration tests of Java applications with CrateDB.*
+
+
+*****
+About
+*****
+
+Introduction
+============
+
+`Testcontainers for Java`_ is a Java library that supports JUnit tests,
+providing lightweight, throwaway instances of common databases suitable
+for integration testing scenarios.
+
+The `Testcontainers CrateDB Module`_ will provide your application test
+framework with a single-node CrateDB instance. You will be able to choose
+the `CrateDB OCI image`_ variant by version, or by selecting the ``nightly``
+release.
+
+What's inside
+=============
+
+This directory includes different canonical examples how to use those
+components within test harnesses of custom applications. Currently,
+all test cases are based on JUnit 4.
+
+
+*****
+Usage
+*****
+
+1. Make sure Java 17 is installed.
```

Comment:
17 or later?

Reply:
Addressed with b5018b402f.

Reply:
I've reverted to only `17` as gradle 4.6 won't work with later versions, for example java 20, so It's better not to confuse the users.